packages/documentation/src/stories/components/select/select.docs.mdx

```diff
@@ -52,6 +51,19 @@ But note that the `<select>` must come first, so we can ensure the correct style
 
 ### Sizing
 
+<div className="alert alert-warning">
+  <h4 className="alert-heading">
+    We're deprecating the regular and medium sizes for text input, textarea and select.
+  </h4>
+  <div className="mt-0">
+    This will make it easier to select the appropriate size variant: small for internal
+    applications, large for external applications.
+    <br />
+    `.form-select-rg` and `.form-select-md` are deprecated and will be removed in the next major
+    version.
+  </div>
+</div>
```

Comment:
```suggestion
{/* prettier-ignore */}
<div className="alert alert-warning">
  <h4 className="alert-heading">We're deprecating the regular and medium sizes for text input, textarea and select.</h4>
  <p className="mt-0">This will make it easier to select the appropriate size variant: small for internal applications, large for external applications.<br />`.form-select-rg` and `.form-select-md` are deprecated and will be removed in the next major version.</p>
</div>
```